python_modules/libraries/dagster-snowflake/dagster_snowflake/resources.py

```diff
@@ -456,7 +479,8 @@ def get_query_status(snowflake: SnowflakeResource, query_id):
             conn.close()
             engine.dispose()
         else:
-            conn = snowflake.connector.connect(**self._connection_args)
+            conn_args = self._connection_args
```

Comment:
If `conn_args` is not reused anywhere else, it would be better to pass `**self._connection_args` directly to `connect()`.